src/queues.c

```diff
@@ -143,9 +143,12 @@ static bool queues_notification_is_finished(struct notification *n, struct dunst
 
         bool is_idle = status.fullscreen ? false : status.idle;
 
+        /* There is no way to detect if the user is idle
+         * on xwayland, so assume they aren't */
+        if (is_running_xwayland()) is_idle = false;
```

Comment:
Since this is only used to disable idle support, I guess it'd be much cleaner/easier to modify the settings struct to disable it on output creation (if wayland is detected).

Reply:
Btw, it's only xwayland. Idle management on Wayland works atm (if your compositor supports it).

Reply:
I agree that's simpler (until we need to know if we're running xwayland elsewhere). I changed it.